tests/queries/0_stateless/03266_with_fill_staleness.reference

```diff
@@ -0,0 +1,139 @@
+add samples
+regular with fill
+2016-06-15 23:00:00	0	original
+2016-06-15 23:00:01	0	
+2016-06-15 23:00:02	0	
+2016-06-15 23:00:03	0	
+2016-06-15 23:00:04	0	
+2016-06-15 23:00:05	5	original
+2016-06-15 23:00:06	5	
+2016-06-15 23:00:07	5	
+2016-06-15 23:00:08	5	
+2016-06-15 23:00:09	5	
+2016-06-15 23:00:10	10	original
+2016-06-15 23:00:11	10	
+2016-06-15 23:00:12	10	
+2016-06-15 23:00:13	10	
+2016-06-15 23:00:14	10	
+2016-06-15 23:00:15	15	original
+2016-06-15 23:00:16	15	
+2016-06-15 23:00:17	15	
+2016-06-15 23:00:18	15	
+2016-06-15 23:00:19	15	
+2016-06-15 23:00:20	20	original
+2016-06-15 23:00:21	20	
+2016-06-15 23:00:22	20	
+2016-06-15 23:00:23	20	
+2016-06-15 23:00:24	20	
+2016-06-15 23:00:25	25	original
+staleness 1 seconds
+2016-06-15 23:00:00	0	original
+2016-06-15 23:00:05	5	original
+2016-06-15 23:00:10	10	original
+2016-06-15 23:00:15	15	original
+2016-06-15 23:00:20	20	original
+2016-06-15 23:00:25	25	original
+staleness 3 seconds
+2016-06-15 23:00:00	0	original
+2016-06-15 23:00:01	0	
+2016-06-15 23:00:02	0	
+2016-06-15 23:00:05	5	original
+2016-06-15 23:00:06	5	
+2016-06-15 23:00:07	5	
+2016-06-15 23:00:10	10	original
+2016-06-15 23:00:11	10	
+2016-06-15 23:00:12	10	
+2016-06-15 23:00:15	15	original
+2016-06-15 23:00:16	15	
+2016-06-15 23:00:17	15	
+2016-06-15 23:00:20	20	original
+2016-06-15 23:00:21	20	
+2016-06-15 23:00:22	20	
+2016-06-15 23:00:25	25	original
+descending order
+2016-06-15 23:00:25	25	original
+2016-06-15 23:00:24	25	
+2016-06-15 23:00:20	20	original
+2016-06-15 23:00:19	20	
+2016-06-15 23:00:15	15	original
+2016-06-15 23:00:14	15	
+2016-06-15 23:00:10	10	original
+2016-06-15 23:00:09	10	
+2016-06-15 23:00:05	5	original
+2016-06-15 23:00:04	5	
+2016-06-15 23:00:00	0	original
+staleness with to and step
+2016-06-15 23:00:00	0	original
+2016-06-15 23:00:03	0	
```

Comment:
Why we do not see next record for step = 3, `2016-06-15 23:00:06` which is still in interval `7`?

Reply:
Because it should be after 5(original row). We should not break the sort order